src/lib/handlebars-converter/handlebars-helpers.spec.js

```diff
@@ -104,8 +104,12 @@ describe('Handlebars helpers', function () {
         { f: 'addHyphensSSN', in: [undefined], out: "" },
         { f: 'addHyphensDate', in: [undefined], out: "" },
         { f: 'formatAsDateTime', in: [undefined], out: "" },
+        {f: 'formatAsDateTime', in: ["20041wrong"], out: null}, // eslint-disable-line
+        {f: 'formatAsDateTime', in: ["2004"], out: "2004"}, // eslint-disable-line
+        {f: 'formatAsDateTime', in: ["20041"], out: "20041"}, // eslint-disable-line
```

Comment:
"2004wrong" and "20041" (both incorrect) should be treated in same manner (preferably return empty). @ranvijaykumar

Reply:
we need to have the same behavior from addHyphensDate too: either return input string or empty.

Reply:
I think the user needs to be informed that there is a problem with the conversion. With our current framework I think an error is the best solution, but it might be worth thinking about a way to return a list of warnings so the user has the choice to continue or not.

Reply:
Yes, I think so. I think it's necessary to validated the input and output data and prompted the user in a unified way for the errors in the conversion process.